frontend/src/components/collab/chat/ChatBubble.tsx

```diff
@@ -0,0 +1,28 @@
+import ChatMessage from "@/types/chat_message";
+
+interface ChatBubbleProps {
+  message: ChatMessage;
+  isSelf: boolean;
+}
+
+const ChatBubble = ({ message, isSelf }: ChatBubbleProps) => {
+  return (
+    <li
+      key={message.uuid}
```

Comment:
Hmm, if you just wanted to generate a unique key, you can consider using `crypto.randomUUID()`, and this `crypto` package is already an existing nodejs API, so you don't need to install any additional package.

Reply:
Ahh, but apparently we can't use the built-in `crypto` package in `client` pages :(